modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/inputflow/JMSTestCase.java

```diff
@@ -179,30 +175,32 @@ public void jmsMapTestWithCustomMappingScenario() throws Exception {
         int startERCount = eventReceiverAdminServiceClient.getActiveEventReceiverCount();
         int startEPCount = eventPublisherAdminServiceClient.getActiveEventPublisherCount();
 
-        //Add StreamDefinition
-        String streamDefinitionAsString = getJSONArtifactConfiguration(samplePath, "org.wso2.event.sensor.stream_1.0.0.json");
+        // Add StreamDefinition.
+        String streamDefinitionAsString = getJSONArtifactConfiguration(samplePath,
+                "org.wso2.event.sensor.stream_1.0.0.json");
```

Comment:
Can we define the org.wso2.event.sensor.stream_1.0.0.json as a constant because it has been referred to in many places.